subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -29,16 +29,19 @@ import (
 	"github.com/microsoft/hivedscheduler/pkg/internal"
 	core "k8s.io/api/core/v1"
 	"k8s.io/klog"
+	"math"
 	"sync"
 )
 
 type HivedAlgorithm struct {
-	// intra-VC scheduler for each VC
+	// scheduler in each VC
 	vcSchedulers map[api.VirtualClusterName]intraVCScheduler
+	// scheduler for opportunistic pods
+	opportunisticSchedulers map[CellChain]*topologyAwareScheduler
```

Comment:
Comment why opportunistic use a different scheduler?
Why it hard codes to be topologyAwareScheduler?